docs/policy/community-testing.md

```diff
@@ -0,0 +1,36 @@
+OSG Community Software Testing
+==============================
+
+Introduction
+------------
+
+In the past, the OSG utilized internal staff for the bulk of software testing.
+Interested external administrators also provided testing when their specialized installation was required.
+
+More recently, we've been employing an ad hoc community testing program.
```

Comment:
Nitpick: since this is a policy doc and merits a certain level of formality, let's stay away from contractions if possible, i.e. `we have been` instead of `we have`

Reply:
Fixed